### PR TITLE
Reduce the manager producer footprint

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>
Reference: https://issues.redhat.com/browse/ACM-10642

image

### CODE REVIEW

File: pkg/transport/config/confluent_config.go [hunk 1]
```diff
@@ -22,6 +22,8 @@ func GetConfluentConfigMap(kafkaConfig *transport.KafkaConfig, producer bool) (*
 		"ssl.endpoint.identification.algorithm": "none",
 	}
 	if producer {
+		_ = kafkaConfigMap.SetKey("go.events.channel.size", 1000)
```
Comment (clyang82) [inline pkg/transport/config/confluent_config.go:25]: What is the default value?

**Review by clyang82 — APPROVED** [review]
LGTM

File: pkg/transport/config/confluent_config.go (hunk 1 above)
Comment (yanmxa) [reply pkg/transport/config/confluent_config.go:25]: The default is `1000000`

**Pull request merged**